root/components/RelationshipsTable.js

```diff
@@ -296,7 +296,6 @@ const RelationshipsTable = ({
         ...PagedLinkTypeGroupT,
         +phrase: string,
       }>> = Object.values(targetTypeGroup)
-        // $FlowIgnore[incompatible-call]
         .map((group: PagedLinkTypeGroupT) => ({
```

Comment:
Is the type specification needed here, or is it just left for clarity, or you just forgot to remove it?

Reply:
Same comment as above. Not needed in the old inference mode, but not certain it's not needed in the new one yet.